doc/nrf/libraries/others/esb.rst

```diff
@@ -10,7 +10,7 @@ Enhanced ShockBurst
 Enhanced ShockBurst (ESB) is a wireless protocol that supports two-way data packet communication including packet buffering, packet acknowledgment, and automatic retransmission of lost packets.
 
 See the :ref:`ESB user guide <ug_esb>` for more information.
-Also, check out the :ref:`esb_prx_ptx` sample.
+Also, check out the :ref:`esb_ptx` and :ref:`esb_prx` samples.
```

Comment:
To me, **check out** sounds a bit too colloquial. Maybe just "see also".

Reply:
Applied (opening new PR soon)